package.json

```diff
@@ -17,30 +17,35 @@
     "@girder/oauth-client": "^0.8.0",
     "@mdi/font": "^6.5.95",
     "@sentry/vue": "^6.16.1",
+    "@types/geojson": "^7946.0.8",
     "axios": "^0.24.0",
+    "cesium": "^1.91.0",
     "django-s3-file-field": "^0.3.0",
     "vue": "^3.2.25",
+    "vue-docgen-api": "^4.44.23",
```

Comment:
Is this used? I don't see any references to it in the code, unless I'm missing something

Reply:
Not explicitly yet. It will be hooked up with storybook to generate component documentation. It is likely a dependency of some of the other storybook stuff I added but I definitely plan on using it directly.